src/Common/src/Internal/Cryptography/BCryptAlgorithmCache.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
It looks like your most recent update added a copy of this file but didn't delete this one.